app/api/role_invites.py

```diff
@@ -135,6 +135,11 @@ def before_update_object(self, role_invite, data, view_kwargs):
         if not has_access('is_organizer', event_id=role_invite.event_id) and (len(list(data.keys())) > 1 or
                                                                               'status' not in data):
             raise UnprocessableEntity({'source': ''}, "You can only change your status")
+        if 'deleted_at' in data and data['deleted_at']:
```

Comment:
Why not simply do `data.get('deleted-at')`

Reply:
Updated it @iamareebjamal